framework/src/base/ComputeNodalKernelsThread.C

```diff
@@ -0,0 +1,85 @@
+/****************************************************************/
```

Comment:
I think we already have a ticket in there for that, we just need to dig it up and get to it.